### ISSUE TITLE
"Create Duplicate" after saving

### ISSUE THREAD

**Issue by lardnicus** [statamic/ideas#594]
The "After Saving" option next to the "Save and Publish" button is honestly a lifesaver... But a great feature for data entry on items that are very similar would be "Create Duplicate"...

So it would essentially save one entry, and then pre-populate all the fields with what you just entered.

**Comment by lardnicus** [comment]
"Create Similar"

**Comment by j3ll3yfi5h** [comment]
Have you already tried the Duplicator Addon?
https://statamic.com/addons/double-three-digital/duplicator

**Comment by lardnicus** [comment]
Yes, I have that installed, but it doesn't have the function I've described...

I know it may seem like a small issue, but for data entry with blueprints with more than 5 fields lets say, that small UX change quickly becomes a huge quality of life outcome for data entry.

**Comment by j3ll3yfi5h** [comment]
Ah, I understand now! This could also be a feature request for the addon, because it is the same functionality on a different position, right?
